acceptancetests/repository/bundles-lxd-profile.yaml

```diff
@@ -0,0 +1,23 @@
+series: bionic
+machines:
+  '0': {}
+  '1': {}
+  '2': {}
+  '3': {}
+applications:
+  lxd-profile:
+    charm: cs:~juju-qa/bionic/lxd-profile-without-devices-0
+    num_units: 4
```

Comment:
Can we make the unit count 8 and deploy the other 4 to the machines.  More than 1 unit per machine seems very common in our failure scenarios

Reply:
Yes, good shout